spec/support/rails_app/Gemfile

```diff
@@ -25,4 +25,4 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'railway-ipc', path: '../../..'
-gem 'google-protobuf', '~> 3.9'
+gem 'google-protobuf', '~> 3.20.0'
```

Comment:
This is a pretty big jump in the protobuf library, albeit they're minor versions, and semantic versioning usually states that minor versions include new functionality,  but were there any degradation messages?

Reply:
Had no output about deprecations and no impact on unit tests. 3.20 is when Arm64 native support appeared in PB so it'll become increasingly important as more folks upgrade in the future.